frontend/packages/dev-console/src/components/helm/form/HelmChartVersionDropdown.tsx

```diff
@@ -94,19 +95,22 @@ const HelmChartVersionDropdown: React.FunctionComponent<HelmChartVersionDropdown
         if (ignore) return;
       }
       if (ignore) return;
-      setHelmChartEntries(json?.entries?.[chartName]);
-      setHelmChartVersions(getChartVersions(json?.entries?.[chartName]));
+      const chartEntries = getChartEntriesByName(json?.entries, chartName, chartRepoName);
+      setHelmChartEntries(chartEntries);
+      setHelmChartVersions(getChartVersions(chartEntries));
     };
     fetchChartVersions();
     return () => {
       ignore = true;
     };
-  }, [chartName]);
+  }, [chartName, chartRepoName]);
 
   const onChartVersionChange = (value: string) => {
-    const chartURL = getChartURL(helmChartEntries, value);
+    const [version, repoName] = value.split('--');
 
-    setFieldValue('chartVersion', value);
+    const chartURL = getChartURL(helmChartEntries, version, repoName);
+
+    // setFieldValue('chartVersion', value);
```

Comment:
Uncomment this or remove it if not needed

Reply:
Done.